### PR TITLE
Cpm v1.2.0

### LINKED ISSUE #34: use the CPM contract version 1-1-3

### PR DESCRIPTION
**Pull request by nbuffon**
New features
-----------------

* **schema**: Provide CPM schema version 1.2.0
* **rust**: apply changes to the data concerned structures: `CollectivePerceptionMessage`, `PerceivedObject`

Closes #34 

How to test
---------------

1. Check the schema validity
    ```
    schema/validate schema/cpm_schema_1-2-0.json
    ```
    **=> it must be valid JSON**
    **=> it must be valid against its own schema `"https://json-schema.org/draft/2020-12/schema"`**
2. Build and run unit tests
    ```
    cargo clean; cargo build --all; cargo test --all
    ```
    **=> Build must be successfull**
    **=> All tests must pass**
3. Run the example application
	 ```
	 cargo run --bin its-client -- --mqtt-host france1 --mqtt-port 11883 --mqtt-username <username> --mqtt-password <password> --mqtt-client-id ora_itsclient_test
	 ```
	 **=> Binary must run and behave correctly**

### CODE REVIEW

**Comment by nbuffon** [comment]
@tigroo Ready for review

File: schema/cam_schema_1-1-2.json [hunk 1]
```diff
@@ -120,7 +120,7 @@
                 },
                 "longitude": {
                   "type": "integer",
-                  "description": "Unit: 0,1 microdegree. oneMicrodegreeEast (10), oneMicrodegreeWest (-10), unavailable(1800000001)",
+                  "description": "Unit: 0.1 microdegree. oneMicrodegreeEast (10), oneMicrodegreeWest (-10), unavailable(1800000001)",
```
Comment (tigroo) [inline schema/cam_schema_1-1-2.json:123]: We've got a a mix of 0.1 and 0,1 separator in all the files . Why just this?

**Review by tigroo — APPROVED** [review]
Tests ok, code rewieved.

File: schema/cam_schema_1-1-2.json (hunk 1 above)
Comment (nbuffon) [reply schema/cam_schema_1-1-2.json:123]: As discussed I'll open an issue about checking all schemas for commas to be replaced by dots

**Comment by nbuffon** [comment]
Just repushed all the same commits unchanged but with my signature in the commit description

**Pull request merged**